tests/test_routes.py

```diff
@@ -808,7 +808,7 @@ def test_route_sub_criteria_scoring_inaccessible_to_commenters(
         ), "Commenter should receive a 302 to authenticator when trying to access the sub criteria scoring page"
         assert (
             response.location
-            == "https://authenticator/service/user?roles_required=TF_LEAD_ASSESSOR|TF_ASSESSOR"  # noqa
+            == "https://authenticator/service/user?roles_required=TF_LEAD_ASSESSOR%7CTF_ASSESSOR"  # noqa
```

Comment:
better to check the response.location creation logic whether it converts into base64

Reply:
This is from a change in utils, I believe: https://github.com/communitiesuk/funding-service-design-utils/commit/2c68a71654699eccebe27b896778ae1f1c29520b

Reply:
I will vaguely leave it to you as to whether this is a bug or not, as I don't have the immediate context to tell. If it is, it probably needs resolving on utils I guess?

Reply:
This is the correct way of the URL I ment https://authenticator/service/user?roles_required=TF_LEAD_ASSESSOR%7CTF_ASSESSOR so we'll have this change in the men time have you tested assessment  all the tests ?

Reply:
All I have done is raise the PR and done the code changes. I haven't really done any QA/etc and I don't have the services running locally, so I'd recommend doing whatever testing before deciding whether to accept this.

I've only raised this PR since I added a breaking change to utils, and _eventually_ you'll probably want to pull that in, so this is me flagging for your awareness and trying to get the ball rolling at least.